libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/barChart/tests/PluggableBarChart.test.tsx

```diff
@@ -162,5 +172,86 @@ describe("PluggableBarChart", () => {
             expect(sortConfig.supported).toBeTruthy();
             expect(sortConfig.disabled).toBeFalsy();
         });
+
+        it("should provide measureSort as default sort for 1M + 1 VB", async () => {
+            const chart = createComponent(defaultProps);
+
+            const sortConfig = await chart.getSortConfig(referencePointMocks.oneMetricOneCategory);
+
+            expect(sortConfig.currentSort).toEqual([
```

Comment:
What about simplify this file by `expect(sortConfig.currentSort).toMatchSnapshot();`?